tools/apigee-envoy-quickstart/scripts/setup-standalone-envoy.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+chmod 644 "$CLI_HOME"/config.yaml
+
+sleep 5
+
+if [[ -z $PIPELINE_TEST ]]; then
+
+  nohup docker run \
+  -v "$CLI_HOME"/config.yaml:/config.yaml \
+  -p 5000:5000 google/apigee-envoy-adapter:v2.0.5 &
```

Comment:
Implications of hardcoding theses tags/versions here?

Could it be omitted?
Should it be defined separately at the top?
Use `latest`?